lms/djangoapps/email_marketing/tests/test_signals.py

```diff
@@ -0,0 +1,169 @@
+"""Tests of email marketing signal handlers."""
+import logging
+
+from django.test import TestCase
+from django.test.utils import override_settings
+from mock import patch
+from util.json_request import JsonResponse
+
+from email_marketing.signals import handle_unenroll_done, \
+    email_marketing_register_user, \
+    email_marketing_user_field_changed
+from email_marketing.tasks import update_user, update_user_email
+from email_marketing.models import EmailMarketingConfiguration
+from django.test.client import RequestFactory
+from student.tests.factories import UserFactory, UserProfileFactory
+
+from sailthru.sailthru_client import SailthruClient
+from sailthru.sailthru_response import SailthruResponse
+from sailthru.sailthru_error import SailthruClientError
+
+log = logging.getLogger(__name__)
+
+
+def update_email_marketing_config(enabled=False, key='badkey', secret='badsecret', new_user_list='new list',
+                                  template='Activation'):
+    """
+    Enable / Disable Sailthru integration
+    """
+    EmailMarketingConfiguration.objects.create(
+        enabled=enabled,
+        sailthru_key=key,
+        sailthru_secret=secret,
+        sailthru_new_user_list=new_user_list,
+        sailthru_activation_template=template
+    )
+
+
+class EmailMarketingTests(TestCase):
+    """
+    Tests for the EmailMarketing signals and tasks classes.
+    """
+
+    def setUp(self):
+        self.request_factory = RequestFactory()
+        self.user = UserFactory.build(username='test', email='test@edx.org')
+        self.profile = UserProfileFactory.build(user=self.user)
+        self.request = self.request_factory.get("foo")
+        update_email_marketing_config(enabled=True)
+        super(EmailMarketingTests, self).setUp()
+
+    def test_is_enabled(self):
+        """
+        Verify that is_enabled() returns True when sailthru integration is enabled.
+        """
+        is_enabled = EmailMarketingConfiguration.current().enabled
+        self.assertTrue(is_enabled)
+
+        config = EmailMarketingConfiguration.current()
+        config.enabled = False
+        config.save()
+        is_not_enabled = EmailMarketingConfiguration.current().enabled
+        self.assertFalse(is_not_enabled)
```

Comment:
This test is really just checking that `ConfigurationModels` work correctly, so I'm not sure there's a huge benefit to it.